src/feature/catalog/api/map-products.ts

```diff
@@ -0,0 +1,19 @@
+import type { CommercetoolsProduct, Product } from '../types';
+
+export const mapToProduct = (product: CommercetoolsProduct): Product => {
+  const currentData = product.masterData?.current;
+  const name = currentData?.name ? Object.values(currentData.name)[0] : 'No name';
+  const imageUrl = currentData?.masterVariant?.images?.[0]?.url;
+
+  const description = currentData?.description
+    ? Object.values(currentData.description)[0]
+    : 'No description available';
+
+  console.log('Raw product data:', product);
+  return {
+    id: product.id,
+    name,
+    description,
+    image: imageUrl || '/placeholder-product.jpg',
+  };
+};
```

Comment:
```suggestion
import type { Product } from '@commercetools/platform-sdk';

export const mapToProduct = (product: Product) => {
  const current = product.masterData.current;
  const name = current.name['en-GB'] || 'no-name';
  const image = current.masterVariant.images?.[0]?.url || '/placeholder-product.jpg';
  const description = current.description?.['en-GB'] || '';

  return {
    id: product.id,
    name,
    description,
    image,
  };
};
```

немношко поигрался со шрифтами, теперь выглядит гораздо читабильнее. Но я бы ещё добавил цену, чтобы она отображалась в каталоге! 

Также обнаружил, что в твоей реализации `fetch-products` идёт обращение к ручке `products`, а в моей `productProjections` иии использование `products` разумнее)

Reply:
Спасибо!
Да, цена просто в след задании поэтому пока не стала тут писать